go.mod

```diff
@@ -4,6 +4,7 @@ module github.com/cosmos/cosmos-sdk
 
 require (
 	github.com/99designs/keyring v1.1.6
+	github.com/DataDog/zstd v1.4.5 // indirect
```

Comment:
Why are there go.mod change?

Reply:
I just updated Tendermint dependency to point to the branch that was created having the fixes, since no release was created. Then running `go mod tidy` the `go.mod` file changed.